_posts/2020-07-16-building-drivers-on-zephyr.md

```diff
@@ -0,0 +1,591 @@
+---
+title: "Tips on Building Drivers for Zephyr"
+description: "A guide on how to build custom drivers for hardware on Zephyr."
+author: jaredwolff
+---
```

Comment:
The cover photo will default to the Interrupt one if another is not given. https://interrupt.memfault.com/blog/img/interrupt-cover-1200px.png. Feel free to add a custom one if you like!

Reply:
Done.